Lib/pathlib.py

```diff
@@ -23,6 +23,14 @@
 # Internals
 #
 
+_WIN_EXT_NAMESPACE_PREFIX = '\\\\?\\'
+_WIN_DRIVE_LETTERS = set('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ')
```

Comment:
I assume this is meant to be immutable?

```suggestion
_WIN_DRIVE_LETTERS = frozenset(string.ascii_letters)
```